cblite/llm/Model.hh

```diff
@@ -0,0 +1,40 @@
+//
+// Model.hh
```

Comment:
Name is probably not descriptive enough given that you can also choose "models" within these "Models"

Perhaps something like LLMProvider